Entity/PostManager.php

```diff
@@ -96,7 +96,10 @@ public function getPager(array $criteria, $page, $limit = 10, array $sort = arra
         $query = $this->getRepository()
             ->createQueryBuilder('p')
             ->select('p, t')
+            ->andWhere('p.publicationDateStart <= :now')
             ->orderBy('p.publicationDateStart', 'DESC');
+        
+        $parameters['now'] = new \DateTime();
```

Comment:
I also don't like using the "mysql" time. 
What if database and application are on different servers, with different timezones. (not common, but possible)